addons/web/static/src/views/fields/statusbar/statusbar_field.js

```diff
@@ -179,11 +183,15 @@ StatusBarField.isEmpty = (record, fieldName) => {
     return record.model.env.isSmall ? !record.data[fieldName] : false;
 };
 StatusBarField.extractProps = ({ attrs, field }) => {
+    const modifiers =
+        typeof attrs.modifiers === "string" ? JSON.parse(attrs.modifiers || "{}") : attrs.modifiers;
```

Comment:
you can use evalDomain to evaluate the readonly modifier (see in field.js)

Reply:
I've already looked into it but I'm not sure how I should get an evalContext for the evalDomain? Also getting the actual value of required is not really needed, I just want to know if it is set to something, that said I'm not sure it's the best approach.